### PR TITLE
Sourcery Refractor

### PR DESCRIPTION
**Pull request by RGBCube**
## Summary

Sourcery AI refractored

This should theoretically not break anything

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by EnokiUN** [comment]
why did mypy break lmfao

**Comment by EnokiUN** [comment]
because it broke some code, that's why

**Comment by EnokiUN** [comment]
Ill have to say no to this one

**Comment by RGBCube** [comment]
> why did mypy break lmfao

it just removed some typings

File: voltage/file.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def __init__(
         else:
             raise TypeError("f must be a string or bytes")
 
-        filename = filename if not filename is None else "file"
+        filename = filename if filename is not None else "file"
```
Comment (EnokiUN) [inline voltage/file.py:71]: also wtf is this

Comment (RGBCube) [reply voltage/file.py:71]: `is not` is better than `not x is y`, makes it much more readable

**Pull request closed without merging**